### PR TITLE
Adds Growth plan template

### PR DESCRIPTION
**Pull request by tobie**
This is a first path at creating a growth plan template for Growth stage projects wanting to join Impact stage.

### CODE REVIEW

**Comment by tobie** [comment]
Suggestion from @mhdawson to have the plan at the bottom and link to it from the table.

**Review by mhdawson — APPROVED** [review]
LGTM +1 from me once my suggestion is incorporated

**Comment by tobie** [comment]
From CPC meeting today:

- Move this PR to project-status
- Add process in the readme which includes:
    - Projects define their desired status themselves.
    - Then get CPC approval for the roadmap

**Review by boneskull — APPROVED** [review]
yes good perfect

**Review by eemeli — APPROVED** [review]
I like this, but shouldn't it go in the project-status repo rather than here?

**Review by kborchers — APPROVED** [review]

**Review by dmethvin — APPROVED** [review]

**Review by christian-bromann — APPROVED** [review]

**Review by joesepi — APPROVED** [review]

**Comment by joesepi** [comment]
Related: #658 
Related: #613

**Review by sendilkumarn — APPROVED** [review]

**Review by designMoreWeb — APPROVED** [review]
Looks good to me 👍🏽

**Comment by joesepi** [comment]
I moved the template to the project-status repo: https://github.com/openjs-foundation/project-status/pull/56

Still need:
* Add process in the readme which includes:
  * Projects define their desired status themselves.
  * Projects ask for CPC approval of the roadmap.

File: GROWTH_PLAN_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# ${PROJECT} Growth Plan
+
+_Note: the purpose of the growth plan is to provide a roadmap for a project to move from Growth stage to Impact stage._
```
Comment (christian-bromann) [inline GROWTH_PLAN_TEMPLATE.md:3]: Based on discussions from #650
```suggestion
_Note: the purpose of the growth plan is to provide a roadmap for a project to move from At Large stage to Impact stage._
```

Comment (christian-bromann) [reply GROWTH_PLAN_TEMPLATE.md:3]: `Growth` is mentioned couple of times more in the following paragraphs

File: GROWTH_PLAN_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## Growth stage requirements
+
+_Note: in theory, Growth stage requirements should already be met by all Growth stage projects.
+As requirements might have changed after a project was accepted at the Growth stage, this isn't necessarily always the case.
+So making sure that all Growth requirements are met is a good first step in any Growth plan._
+
+| Requirement | Current status | Desired status | Plan | Timeline | Completed |
+| --- | ---| --- | --- | --- | --- | 
+| Successfully used in production by at least two independent end users. | {CURRENT_STATUS} | 2 | {PLAN} | {DATE} | <!--✅--> |
```
Comment (christian-bromann) [inline GROWTH_PLAN_TEMPLATE.md:13]: I would suggest to generalise this a bit more. Production usage might work for Node.js but not for other Node.js based tools.

```suggestion
| Project shows substantial adoption in production environments or usage within the ecosystem | {CURRENT_STATUS} | 2 | {PLAN} | {DATE} | <!--✅--> |
```

File: GROWTH_PLAN_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+| Sufficient level of community participation to meet goals outlined in growth plan. | {CURRENT_STATUS} | {DESIRED_STATUS} | {PLAN} | {DATE} | <!--✅--> |
+| Accepts security reports and publicly disclose them after the fix has been made available. | {CURRENT_STATUS} | {DESIRED_STATUS} | {PLAN} | {DATE} | <!--✅--> |
+
+## Impact stage requirements
+
+| Requirement | Current status | Desired status | Plan | Timeline | Completed |
+| --- | ---| --- | --- | --- | --- | 
+| Governing body of >= 5 members of which no more than 1/3 is affiliated with the same employer. | {CURRENT_STATUS} | {DESIRED_STATUS} | {PLAN} | {DATE} | <!--✅--> |
+| Publicly documented governance, decision-making, contribution, and release processes. | {CURRENT_STATUS} | {DESIRED_STATUS} | {PLAN} | {DATE} | <!--✅--> |
+| Healthy number of committers from at least two organizations. | {CURRENT_STATUS} | {DESIRED_STATUS} | {PLAN} | {DATE} | <!--✅--> |
```
Comment (christian-bromann) [inline GROWTH_PLAN_TEMPLATE.md:24]: This suggestion might better emphasise the importance of ongoing contributions.

```suggestion
| Healthy number of regular committers from at least two organizations. | {CURRENT_STATUS} | {DESIRED_STATUS} | {PLAN} | {DATE} | <!--✅--> |
```

**Review by christian-bromann — COMMENTED** [review]
Some suggestions

**Comment by eemeli** [comment]
@christian-bromann You'll probably want to add your suggestions to openjs-foundation/project-status#56, where this PR has been "moved".

@tobie / @joesepi Perhaps this PR should be closed, to avoid more duplication?

**Comment by tobie** [comment]
Closed in favor of openjs-foundation/project-status#56.

**Pull request closed without merging**